packages/web-twig/src/Resources/components/Collapse/Collapse.twig

```diff
@@ -11,17 +11,17 @@
 {%- set _contentClassName = _spiritClassPrefix ~ 'Collapse__content' -%}
 
 {# Attributes #}
-{%- set _idAttr = _id ? 'id=' ~ _id ~ '' : null -%}
-{%- set _breakpointAttr = _breakpoint ? 'data-breakpoint=' ~ _breakpoint ~ '' : null -%}
-{%- set _parentAttr = _parent ? 'data-parent=' ~ _parent ~ '' : null -%}
+{%- set _idAttr = _id ? 'id="' ~ _id ~ '"' : null -%}
+{%- set _breakpointAttr = _breakpoint ? 'data-breakpoint="' ~ _breakpoint ~ '"' : null -%}
```

Comment:
When we are on it, could you please also improve the names of the private `*Attr` variables? I just [described it in another PR](https://github.com/lmc-eu/spirit-design-system/pull/606#discussion_r1064743641).

<img width="819" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/211346141-384d47e7-a02f-40fb-9291-ff62bb16e67e.png">

Reply:
done